default/stringtables/en.txt

```diff
@@ -4081,7 +4087,10 @@ FW_FLEET_STRUCTURE_SUMMARY
 Fleet Structure - Equal to the total structure of ships in the fleet.
 
 FW_FLEET_DAMAGE_SUMMARY
-Fleet Damage - Equal to the total direct fire damage of ships in the fleet.
+Fleet Damage - Equal to the total maximum structural damage of ships and fighters in the fleet.
+
+FW_FLEET_DESTROY_SUMMARY
+Fleet Destruction - Equal to the total maximum fighter destruction of ships and fighters in the fleet.
```

Comment:
I don't understand this description.

What's the difference between "fleet destruction" and "fighter destruction"?

Is it "fighter (destructions of ships and fighters)" or "(fighter destruction) of ships and fighters"?

I think "structural damage" is relatively clear, but what is "destruction"?

Reply:
fleet destruction is to fighter destruction what fleet damage is to structural damage.

i'm not a native speaker, i tried to use destruction only when talking about fighter targets and talking about structural damage when talking about ship targets.
Maybe a clearer term exists. 

Probably fleet destruction could be worded much better, maybe: Fleet Destruction - Equal to the maximal total destruction of fighters by the fleet.
('by the fleet' is probably a germanism)
Or Fleet Destruction - The maximum number of enemy fighters the fleet is able to destroy in a turn.

Reply:
Using words consistently doesn't mean the reader will immediately pick up on the distinction when similar expressions are used with a word substituted but used in a different way. In particular, I think the use of "fighter destruction" to mean fighters destroyed and "fleet destruction" to mean destruction by a fleet is confusing.

"destruction of fighters" and "destruction of fighters by fleet" would be clearer, or just "fighter destruction" and "fighter destruction by fleet" to save a few characters.